Core/GDCore/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -674,7 +690,7 @@ gd::String EventsCodeGenerator::GenerateParameterCodes(
     argOutput =
         GenerateObject(parameter.GetPlainString(), metadata.GetType(), context);
   } else if (metadata.GetType() == "relationalOperator") {
-    argOutput += GenerateRelationalOperatorCodes(parameter.GetPlainString());
+    argOutput += parameter.GetPlainString();

```

Comment:
This is actually used nowhere I guess? Having a relationalOperator "alone"?

Reply:
Yes, I guess unless the instruction is wrongly declared.
```C++
  if (instrInfos.codeExtraInformation.type == "number" ||
      instrInfos.codeExtraInformation.type == "string") {
    predicate = GenerateRelationalOperatorCall(
        instrInfos,
        arguments,
        instrInfos.codeExtraInformation.functionCallName);
  } else {
    predicate = instrInfos.codeExtraInformation.functionCallName + "(" +
               GenerateArgumentsList(arguments, 0) + ")";
  }
```

Reply:
I've checked the codebase and can't find any other usage of relationalOperator apart from CompareStrings/CompareNumbers, which have a custom code generation. So I think we're good.